go/vt/logutil/logger.go

```diff
@@ -231,6 +231,13 @@ func (ml *MemoryLogger) String() string {
 	return buf.String()
 }
 
+// Clear clears the logs.
+func (ml *MemoryLogger) Clear() {
```

Comment:
optional: You could name this `ClearForTesting` to clarify that it should only be used by tests?